superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx

```diff
@@ -56,7 +56,7 @@ const hostField = ({
   <ValidatedInput
     id="host"
     name="host"
-    value={db?.parameters?.host || ''}
+    value={db?.parameters?.host}
```

Comment:
This is a small fix for blocking us from typing values for `Adding new databases`